src/sway.rs

```diff
@@ -74,9 +74,16 @@ pub fn get_all_windows(workspace: &Node) -> Vec<Node> {
     nodes
 }
 
-pub fn focus(conn: Arc<Mutex<Connection>>, con_id: i64) {
+pub fn focus(conn: &Arc<Mutex<Connection>>, con_id: i64) {
     let mut conn_lock = conn.lock().unwrap();
     conn_lock
         .run_command(format!("[con_id={}] focus", con_id))
         .expect("failed to focus container");
 }
+
+pub fn swap(conn: &Arc<Mutex<Connection>>, con_id: i64) {
+    let mut conn_lock = conn.lock().unwrap();
+    conn_lock
+        .run_command(format!("swap container with con_id {}", con_id))
+        .expect("failed to sway container");
```

Comment:
minor typo: I think you mean "failed to *swap* container"

Reply:
Fixed.